src/app/routes/cpsAsset/getInitialData/index.js

```diff
@@ -60,7 +61,24 @@ const transformJson = async (json, pathname) => {
   }
 };
 
-export default async ({ path: pathname, service, variant, pageType }) => {
+// Moved to a different function to additional data processing other than most watched
+const processAdditionalData = ({ data, service, path: pathname, toggles }) => {
+  const processedData = processMostWatched({
+    data,
+    service,
+    path: pathname,
+    toggles,
+  });
+
+  return processedData;
+};
```

Comment:
does this need a wrapping function. Can you not just call `processMostWatched` directly? Not sure what the comment means. Can it be removed too?

Reply:
Yeah, following Ryan's recommendation on this